internal/cmd/policy/listrules/listrules.go

```diff
@@ -10,17 +10,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type options struct{}
+type options struct {
+	targetRef string
+}
 
-func (o *options) AddFlags(_ *cobra.Command) {}
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.targetRef,
+		"targetRef",
+		"",
+		"to specify which ref should be inspected",
+	)
+	cmd.MarkFlagRequired("targetRef") //nolint:errcheck
```

Comment:
does this need to be required? Maybe make the default the main policy ref?

Reply:
Fixed

Reply:
FWIW, I still see this in the latest commit. It possibly snuck back in? :)

Reply:
Fixed